CHANGELOG.md

```diff
@@ -1,5 +1,37 @@
 # Delta Chat Android Changelog
 
+## v1.36.0
+2023-03
+
+* new, easy method of adding a second device to your account:
+  select "Add as a Second Device" after installation and scan a QR code from the old device
+* view "All Media" of all chats by the corresponding option in the chat list's menu
+* add "Clear Chat" option to remove all messages from a chat
+* show non-deltachat emails by default for new installations
+  (you can change this at "Settings / Chats and Media)
+* show notifications for all accounts
+* make better use of dark/light mode in "Show full message"
+* show icon beside info messages of apps
+* resilience against outages by caching DNS results for SMTP connections
+* prefer TLS over STARTTLS during autoconfiguration, set minimum TLS version to 1.2
+* use SOCKS5 configuration also for HTTP requests
+* make invite QR codes even prettier
+* improve storage my free unused areas sooner
```

Comment:
s/my/by/ ???

Reply:
19 Improving storage and releasing unused areas sooner? 🤔